python/lsst/ts/imsim/imsimCmpt.py

```diff
@@ -277,14 +273,15 @@ def addConfigHeader(self, obsMetadata):
             Header information for ImSim config
         """
         headerText = "  header:\n"
-        headerText += f"    mjd: {obsMetadata.mjd}\n"
+        headerText += f"    mjd: *mjd\n"
         headerText += f"    observationStartMJD: {obsMetadata.mjd - (15/(60*60*24))}\n"
-        headerText += f"    seqnum: {obsMetadata.seqNum}\n"
-        headerText += f"    band: {obsMetadata.band}\n"
+        headerText += f"    seqnum: *seqnum\n"
```

Comment:
Out of curiosity, is this using star unpacking of obsMetadata? I am guessing this can't be used for the other fields, right?

Reply:
No, when loading the yaml from a string it will fill in these values with values from `evalVariables`. We could just load them all from the `obsMetadata` object if that's easier to understand.

Reply:
I think it’s fine